docker/witnet-rust/Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM --platform=$TARGETPLATFORM ubuntu:jammy
+FROM --platform=$TARGETPLATFORM ubuntu:noble
 
 # Install basic environment dependencies
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt update && apt install -y --no-install-recommends \
     ca-certificates \
     curl \
-    netcat \
+    netcat-traditional \
     jq
 
 # Clean up apt packages so the docker image is as compact as possible
-RUN apt-get clean && apt-get autoremove
+RUN apt clean && apt autoremove
```

Comment:
It is customary in the GNU world not to use `apt` in scripts. This is explained in its [man page](https://manpages.debian.org/buster/apt/apt.8#SCRIPT_USAGE_AND_DIFFERENCES_FROM_OTHER_APT_TOOLS) and the command will even complain about it when it detects that the current `tty` isn't interactive.